.buildkite/scripts/steps/artifacts/build.sh

```diff
@@ -18,6 +18,11 @@ node scripts/build "$RELEASE_ARG" --all-platforms --debug --docker-cross-compile
 echo "--- Build dependencies report"
 node scripts/licenses_csv_report "--csv=target/dependencies-$VERSION.csv"
 
+echo "--- Extract default i18n messages"
+mkdir -p target/i18n
+node scripts/i18n_extract
+buildkite-agent artifact upload "target/i18n/en.json"
```

Comment:
since we have `buildkite-agent artifact upload "*"` on line 51. Is this still needed?

Reply:
2cbd49c4e312f319a2f0dbe4a0aff2eb23d18ab8